util/container/Dockerfile

```diff
@@ -4,174 +4,127 @@
 
 # Docker container for Snitch development.
 
-# 1. Stage
-FROM ubuntu:22.04 AS builder
-ARG CMAKE_VERSION=3.19.4
-ARG PYTHON_VERSION=3.9.12
+# 1. Stage: Use the verilator container to get the Verilator binary.
+ARG VERILATOR_VERSION=5.020
+ARG UBUNTU_VERSION=22.04
+
+# 1. Stage: Install additional IIS tools
+FROM ubuntu:${UBUNTU_VERSION} AS builder
 ARG BENDER_VERSION=0.27.1
-ARG SPIKE_DASM_VERSION=0.1.0
-ARG VERILATOR_VERSION=5.006
 ARG DOXYGEN_VERSION=1.12.0
-# Run dpkg without interactive dialogue
-ARG DEBIAN_FRONTEND=noninteractive
```

Comment:
I believe it would be better to keep the `DEBIAN_FRONTEND` variable in every stage where we use `apt`

Reply:
It is only used for a single package. Otherwise, the container would not finish building since it expects an input from the user.

Reply:
No, I mean we should not only use where it fails otherwise, but everywhere we use `apt`, as a guarantee that it will never fail, as a consequence of packages requiring user input.

Also, now that I look at this again, use of `apt` in scripts is discouraged, in favour of `apt-get`, see the note at https://ubuntu.com/server/docs/package-management#removing-packages.

Reply:
Addressed in https://github.com/pulp-platform/snitch_cluster/pull/192/commits/25fc2942865c0901c8dce6cb5223355e2ebb871e and https://github.com/pulp-platform/snitch_cluster/pull/192/commits/79a03dffed611c947287a06ed038a4f6e31d9424